src/billing/utils/timeHelpers.test.ts

```diff
@@ -1,9 +1,11 @@
 import {
-  nsToHours,
   hoursToDays,
   hoursToNs,
-  nsToDays,
   minToSeconds,
+  nsToDays,
+  nsToHours,
+  nsToSeconds,
+  secondsToNs,
```

Comment:
Just took the opportunity to alphabetize since I was adding a couple into the import list.

Reply:
heroic!